content/pt-br/multitenancy.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Multitenancy
+status: Completed
+category: propriedade
+---
+
+## O que é
+
+Multitenancy (em inglês, multi-inquilinos) se refere a uma única instalação de software que atende a múltiplos inquilinos. Um inquilino pode ser um usuário, uma aplicação ou um grupo de usuários/aplicações que usam esse software para operar no seu próprio dataset. Esses inquilinos não compartilham dados (a não ser que explicitamente configurado pelo proprietário), e podem até não estar cientes um dos outros.
```

Comment:
"(a não ser que explicitamente configurado pelo proprietário)" How about using "instruído" instead of "configurado"?